code/modules/reagents/reagents/food_drinks.dm

```diff
@@ -1714,6 +1727,110 @@
 	glass_desc = "A glass of refreshing Space Cola"
 	glass_special = list(DRINK_FIZZ)
 
+/datum/reagent/drink/soda/lemon_soda
+	name = "Lemon Soda"
+	id = "lemonsoda"
+	description = "Soda made using lemon concentrate. Sour."
+	taste_description = "strong sourness"
+	reagent_state = LIQUID
+	color = "#ffe658"
+	adj_drowsy = -3
+	adj_temp = -5
+
+	glass_name = "lemon Soda"
+	glass_desc = "A glass of refreshing Lemon Soda. So sour!"
+	glass_special = list(DRINK_FIZZ)
+	allergen_type = ALLERGEN_FRUIT
+
+/datum/reagent/drink/soda/apple_soda
+	name = "Apple Soda"
+	id = "applesoda"
+	description = "Soda made using fresh apples."
+	taste_description = "crisp juiciness"
+	reagent_state = LIQUID
+	color = "#c73737"
+	adj_drowsy = -3
+	adj_temp = -5
+
+	glass_name = "Apple Soda"
+	glass_desc = "A glass of refreshing Apple Soda. Crisp!"
+	glass_special = list(DRINK_FIZZ)
+	allergen_type = ALLERGEN_FRUIT
+
+
+/datum/reagent/drink/soda/straw_soda
+	name = "Strawberry Soda"
+	id = "strawsoda"
+	description = "Soda made using sweet berries."
+	taste_description = "oddly bland"
+	reagent_state = LIQUID
+	color = "#ffa3a3"
+	adj_drowsy = -3
+	adj_temp = -5
+
+	glass_name = "Orange Soda"
```

Comment:
this should be the strawberry soda's stuff

Reply:
Fixed Error